### PR TITLE
Adding flag for checking if running remotely, fixing some issues.

### PR DESCRIPTION
**Pull request by pavithrasv**
1. Adding flag for checking if running remotely
2. Fix error when trying to string replace '-rc' with 'rc' in version.
3. Add 'py3' tag only for TF docker images < 2.1.0 as the later images are by default py3.
4. Add an error and solution to README section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,14 +360,23 @@ Things to keep in mind when running your jobs remotely:
...
+
+**Error like**: Creating a generator within a strategy scope is disallowed, because there is ambiguity on how to replicate a generator (e.g. should it be copied so that each replica gets the same random numbers, or 'split' so that each replica gets different random numbers).
+
+**Solution**: Passing `distribution_strategy='auto'` to `run` API wraps all of your script in a TF distribution strategy based on the cluster configuration provided. You will see the above error or something similar to it, if for some reason an operation is not allowed inside distribution strategy scope. To fix the error, please pass `None` to the `distribution_strategy` param and create a strategy instance as part of your training code as shown in [this](https://github.com/tensorflow/cloud/blob/master/tests/testdata/save_and_load.py) example.
+
+### Version not supported for TPU training
+
+**Error like**: RuntimeError: There was an error submitting the job.Field: tpu_tf_version Error: The specified runtime version '2.3' is not supported for TPU training.  Please specify a different runtime version. See https://cloud.google.com/ml-engine/docs/runtime-version-list for a list of supported versions
```
Comment (fchollet) [inline README.md:373]: Use backticks around code keywords (e.g. `RuntimeError`), use markdown formatting for links

File: README.md [hunk 2]
```diff
@@ -347,14 +360,23 @@ Things to keep in mind when running your jobs remotely:
...
+
+**Solution**: Please use TF version 2.1. See TPU Strategy in [Cluster and distribution strategy configuration section](#cluster-and-distribution-strategy-configuration).
```
Comment (fchollet) [inline README.md:375]: 2.3 doesn't work? 2.2 doesn't work? :(

File: tensorflow_cloud/run.py [hunk 3]
```diff
@@ -28,6 +28,11 @@
 from . import validate
 
 
+def is_running_remotely():
```
Comment (fchollet) [inline tensorflow_cloud/run.py:31]: I suggest `remote()`, so users can do

`if tfc.remote():`

which is shorter and simpler

We could also have the reverse, `local()`

File: README.md (hunk 2 above)
Comment (pavithrasv) [reply README.md:375]: Unfortunately no. They are in the process of updating.

File: tensorflow_cloud/run.py (hunk 3 above)
Comment (pavithrasv) [reply tensorflow_cloud/run.py:31]: Used remote.

File: README.md (hunk 1 above)
Comment (pavithrasv) [reply README.md]: Removed link from here as only the main error message is required.

**Comment by pavithrasv** [comment]
Thank you Francois!

**Pull request merged**